src/main/scala/com/cognite/spark/datasource/TimeSeriesRelation.scala

```diff
@@ -18,12 +18,12 @@ case class TimeSeriesConflict(notFound: Seq[Long])
 case class TimeSeriesNotFound(notFound: Seq[String])
 
 case class TimeSeriesItem(
-    name: String,
-    isString: Boolean,
+    name: Option[String],
+    isString: Option[Boolean],
```

Comment:
Why are we changing this? Makes sense for PostTimeSeriesItem but I don't see how it makes sense here...?

Reply:
SparkSchemaHelper interprets all non-Option fields as required when getting the schema in fromRow. I suppose we could find another way to get UpdateTimeSeriesItem from a row, any suggestions?

Reply:
Could we make it a separate case class and use fromRow with whatever schema we get from that?

Reply:
Definitely